src/main/java/org/springframework/data/redis/connection/jedis/StreamConverters.java

```diff
@@ -37,7 +40,17 @@
 
 import org.jspecify.annotations.Nullable;
 import org.springframework.data.domain.Range;
-import org.springframework.data.redis.connection.RedisStreamCommands;
+import org.springframework.data.redis.connection.RedisStreamCommands.XAddOptions;
```

Comment:
Something seems off with formatting, typically, we use star imports for a import count over 10.